### PR TITLE
SAM-2709 Add template setting for Submission Email

### PR DESCRIPTION
**Pull request by master-bob**
Add in the template setting to control submission emails / instructor
notifications.

### CODE REVIEW

**Comment by bjones86** [comment]
@master-bob does this depend on SAM-2783 (#2047)

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SaveAssessmentSettings.java [hunk 1]
```diff
@@ -194,6 +189,10 @@ public AssessmentFacade save(AssessmentSettingsBean assessmentSettings, boolean
     //log.info("**unlimited submission="+assessmentSettings.getUnlimitedSubmissions());
     //log.info("**allowed="+control.getSubmissionsAllowed());
 
+	if (assessmentSettings.getInstructorNotification() != null) {
+		control.setInstructorNotification(new Integer(assessmentSettings.getInstructorNotification()));
+	}
+
```
Comment (bjones86) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SaveAssessmentSettings.java]: Spaces vs. tabs in this block.

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/TemplateLoadListener.java [hunk 2]
```diff
@@ -152,6 +148,8 @@ public boolean loadAssessment(TemplateBean templateBean, String templateId)
         }
         if (aac.getLateHandling() != null)
           templateBean.setLateHandling(aac.getLateHandling().toString());
+        if (aac.getInstructorNotification() != null)
+          templateBean.setInstructorNotification(aac.getInstructorNotification().toString());
```
Comment (bjones86) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/TemplateLoadListener.java]: I'd prefer to have curly braces on one-line if/else statements. I realize the surrounding code may not, but the precedent should be to include them.

**Comment by bjones86** [comment]
Looks good for the most part, a few minor comments that are style related. My only other thought is that the SQL scripts should be included in the `sakai_12_mysql_conversion.sql` and 'sakai_12_oracle_conversion.sql` as well, since this is going into 12 as well as 11.

**Comment by bjones86** [comment]
Pretty sure the process is to run them sequentially, so leaving them in the 11 conversion files is probably sufficient. You can disregard my previous comment @master-bob :)

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SaveAssessmentSettings.java (hunk 1 above)
Comment (master-bob) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SaveAssessmentSettings.java]: Is fixed in latest.

**Comment by ottenhoff** [comment]
Pretty clean patch ..... looks good to me .... I'm happy with this going in

**Pull request merged**